localstack/services/awslambda/lambda_executors.py

```diff
@@ -638,6 +647,18 @@ def execute_java_lambda(self, event, context, handler, main_file):
         return result, log_output
 
 
+class Util:
+
+    @staticmethod
+    def get_java_opts(port):
+        opts = config.LAMBDA_JAVA_OPTS
+        if opts.find('_debug_port_'):
+            java_opts = opts.replace('_debug_port_', ('address=%s' % port))
```

Comment:
Sorry for being so picky on this PR, but can we change this such that `_debug_port_` only represents the actual port number? I.e., the user would specify `LAMBDA_JAVA_OPTS=...address=_debug_port_`

Reply:
No problem,  it should be the best solution. Ok, let me change it

Reply:
Awesome, thanks